### PR TITLE
TTH-8: Added routing between LandingPage and MatchPage

### PR DESCRIPTION
**Pull request by roseyume**
* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Routing added between website views: Landing Page and MatchPage

* **What is the current behavior? What does the code do before you made the new changes?** 
Displays landing page with static navbar

* **What is the new behavior?**
Navbar routes to the LandingPage through /home link and MatchPage through /match link. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
react-router-dom dependency added. Make sure to rerun "npm install" or "npm install react-router-dom"

* **Other information**:

### CODE REVIEW

File: frontend/src/components/Navbar.js [hunk 1]
```diff
@@ -6,8 +6,7 @@ import {
   NavbarBrand,
   Nav,
   NavItem,
-  NavLink,
-  NavbarText
```
Comment (thomasabigail) [inline frontend/src/components/Navbar.js:10]: Why did you remove NavbarText and not add it back?

Comment (roseyume) [reply frontend/src/components/Navbar.js:10]: I removed importing the NavBarText because it was not being used. I also got notifications from the console that it was being imported but not called.

**Review by thomasabigail — APPROVED** [review]

**Review by areebakq — COMMENTED** [review]
It looks good to me!

**Review by areebakq — APPROVED** [review]
Looks good!
